bindings/rust/bench/src/harness.rs

```diff
@@ -14,27 +14,37 @@ pub fn read_to_bytes(path: &str) -> Vec<u8> {
     read_to_string(path).unwrap().into_bytes()
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Mode {
     Client,
     Server,
 }
 
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
+pub enum HandshakeType {
+    #[default]
+    Full,
```

Comment:
You might just want "ClientAuth, NoClientAuth" as your enums. "Full" isn't going to mean much to anyone.